lib/modules/txt.mjs

```diff
@@ -343,3 +343,51 @@ export async function validateCopyrightNoticeSectionIsNumbered (doc, { mode = MO
 
   return result
 }
+
+/**
+ * Validate submission compliance line
+ *
+ * @param {Object} doc Document to validate
+ * @param {Object} [opts] Additional options
+ * @param {number} [opts.mode=0] Validation mode to use
+ * @returns {Array} List of errors/warnings/comments or empty if fully valid
+ */
+export async function validateSubmissionComplianceLine (doc, { mode = MODES.NORMAL } = {}) {
+  const result = []
+
+  if (!doc.data.contains.submissionCompliance) {
+    result.push(new ValidationError(
+      'SUBMISSION_COMPLIANCE_LINE_MISSING',
+      'Missing notice of Compliance with BCP 78 and BCP 79 according to IETF Trust Provisions of 28 Dec 2009, Section 6.a or Provisions of 12 Sep 2009, Section 6.a',
+      {
+        ref: 'https://trustee.ietf.org/wp-content/uploads/IETF-TLP-4.pdf'
+      }
+    ))
+  }
+
+  return result
+}
+
+/**
+ * Validate submission compliance line page
+ *
+ * @param {Object} doc Document to validate
+ * @param {Object} [opts] Additional options
+ * @param {number} [opts.mode=0] Validation mode to use
+ * @returns {Array} List of errors/warnings/comments or empty if fully valid
+ */
+export async function validateSubmissionComplianceLinePage (doc, { mode = MODES.NORMAL } = {}) {
+  const result = []
+
+  if (!doc.data.possibleIssues.submissionCompliancePage || doc.data.possibleIssues.submissionCompliancePage > 1) {
+    result.push(new ValidationError(
+      'SUBMISSION_COMPLIANCE_LINE_NOT_ON_THE_FIRST_PAGE',
+      'Notice of conformance to BCP 78 and BCP 79 in accordance with IETF Trust Statements, December 28, 2009, Section 6.a or Trust Statements, September 12, 2009, Section 6.a does not appear on the first page',
+      {
+        ref: 'https://trustee.ietf.org/wp-content/uploads/IETF-TLP-4.pdf'
```

Comment:
https://trustee.ietf.org/wp-content/uploads/Corrected-TLP-5.0-legal-provsions.pdf
(And I just spotted the typo in that link, but the typo is real. The trust will be replacing their website soon anyhow, so go with this link for now).